SDWebImageSwiftUI/Classes/ImagePlayer.swift

```diff
@@ -38,7 +38,7 @@ public final class ImagePlayer : ObservableObject {
     @Published public var currentFrame: PlatformImage?
     
     /// Current playing frame index
-    @Published public var currentFrameIndex: UInt = 0
+    public var currentFrameIndex: UInt = 0
```

Comment:
What is the purpose of this change?

Reply:
I see in this callback animationFrameHandler, it updates two published fields and each time will trigger two passes of UI re-rendering, it seems to be inefficient. So I change it to only update once for each gif frame update:

Looks like we only need this currentFrame to render animation.

It's more like some performance optimization.